packages/parser-babel-config/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "ajv": "^6.2.1"
+  },
+  "description": "sonarwhal parser needed to analyze babel config",
+  "devDependencies": {
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint": "^4.18.2",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-markdown": "^1.0.0-beta.7",
+    "eslint-plugin-typescript": "^0.9.0",
+    "markdownlint-cli": "^0.7.0",
+    "npm-link-check": "^2.0.0",
+    "npm-run-all": "^4.1.2",
+    "nyc": "^11.4.1",
+    "proxyquire": "^2.0.0",
+    "rimraf": "^2.6.2",
+    "sonarwhal": "^0.29.0-beta.0",
+    "typescript": "^2.6.2",
+    "typescript-eslint-parser": "^14.0.0"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src",
+    "npm-shrinkwrap.json"
+  ],
+  "homepage": "https://sonarwhal.com/",
+  "keywords": [
+    "sonarwhal",
+    "sonarwhal-parser",
+    "babel-config"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/babel-config.js",
+  "name": "@sonarwhal/parser-babel-config",
+  "nyc": {
+    "extends": "../../../../.nycrc"
+  },
+  "peerDependencies": {
+    "sonarwhal": "^0.29.0-beta.0"
+  },
+  "repository": "sonarwhal/sonarwhal",
+  "scripts": {
+    "build": "yarn clean && npm-run-all build:*",
```

Comment:
Change `yarn` to `npm` in all the `scripts` to avoid issues per 88ee482bba51a8de564c2a2c2d03be9f1e5f7199